trunk/NDHMS/HYDRO_drv/module_HYDRO_drv.F

```diff
@@ -907,8 +907,15 @@ subroutine driveChannelRouting(did)
        RT_DOMAIN(did)%TYPEL, RT_DOMAIN(did)%ORDER, RT_DOMAIN(did)%MAXORDER,   RT_DOMAIN(did)%CH_LNKRT, &
        rt_domain(did)%overland%streams_and_lakes%lake_mask, nlst_rt(did)%DT, nlst_rt(did)%DTCT, nlst_rt(did)%DTRT_CH, &
        RT_DOMAIN(did)%MUSK, RT_DOMAIN(did)%MUSX, RT_DOMAIN(did)%QLINK, &
+       !RT_DOMAIN(did)%CHANLEN, RT_DOMAIN(did)%MannN, RT_DOMAIN(did)%So, RT_DOMAIN(did)%ChSSlp,RT_DOMAIN(did)%Bw,& 
```

Comment:
I commented out the old declarations from NWM v1.2 with channel loss. These can be removed if the new version works.

Reply:
let's go ahead and remove the commented out.  if we need to revert, we have the commit.  @jmccreight may disagree.  im looking for the code to be clean in the PR.